docs/source/cluster_management/sweep.rst

```diff
@@ -31,113 +30,15 @@ Reasons to Sweep
-
-.. _sweep-priority-overrides:
-
-Priority Overrides
-~~~~~~~~~~~~~~~~~~
-
-.. warning::
-   Specifying priority tables can be useful for influencing sweep's behaviour in the short run.
-   However, while configured, no table that is not marked as a priority table will ever be swept, meaning that old versions of cells will not be cleaned up.
-   It is not intended for priority tables to be specified in a steady state, generally speaking.
-
-There may be situations in which the background sweeper's heuristics for selecting tables to sweep may not satisfy one's requirements.
-One can influence the selection process by configuring the ``sweepPriorityOverrides`` parameter in runtime configuration.
-
-.. csv-table::
-   :header: "AtlasDB Runtime Config", "Default", "Description"
-   :widths: 20, 40, 200
-
-   ``blacklistTables``, "[]", "List of fully qualified table names (e.g. ``namespace.tablename``) which the background sweeper should not sweep."
-   ``priorityTables``, "[]", "List of fully qualified table names which the background sweeper should prioritise for sweep."
-
-Note that the conditions for table selection are only checked between iterations of sweep. The algorithm for deciding
-whether a table that is currently being swept should continue to be swept is as follows:
-
-1. If the table being swept is a priority table, continue sweeping it.
-2. If there are any priority tables and the table currently being swept is not a priority table, stop sweeping it and
-   switch to a random priority table.
-3. If the table being swept is not blacklisted, continue sweeping it.
-4. There are no priority tables, and the table being swept is blacklisted; select another table using standard sweep
-   heuristics.
-
-Note that if blacklist/priority settings are subsequently removed, sweep will continue sweeping the table that those
-settings have made it select (in particular, it will not immediately return to the original table it was sweeping).
-
-Also, as we don't know the data distribution of the original table we were sweeping, we don't know how much of the
-table we have swept; we thus don't have enough information to update the sweep priority table with our results while
-registering that we have only done a partial sweep. In practice, this would tend to mean that the probability the
-original table being swept will be picked again for sweeping after overrides have been removed may be a little higher
-than if we accounted for the partial sweep.
+- :ref:`Targeted sweep<targeted-sweep>`:
+  Targeted sweep persists the information about each transactional write as it is performed, and then uses this information to delete all older versions of each of the cells using a single ranged tombstone.
 
 .. toctree::
     :maxdepth: 1
     :hidden:
 
-    sweep/background-sweep
-    sweep/sweep-endpoints
-    sweep/sweep-cli
-    sweep/sweep-logs
-    sweep/freeing-space
+    standard-sweep
+    targeted-sweep
```

Comment:
Freeing Space should be at this level as well - that covers nodetool compact and Oracle's shrink-space etc, which you still need regardless of whether you place tombstones via standard or targeted sweep